demos/azure-resource-optimization-toolkit/azuredeploy.parameters.us.json

```diff
@@ -1,15 +1,18 @@
 {
-    "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
-    "contentVersion": "1.0.0.0",
-    "parameters": {
-      "azureAdminPwd": {
-        "value": "GEN-PASSWORD"
-      },
-      "azureAdmin": {
-        "value": "GEN-UNIQUE"
-      },
-      "location": {
-        "value": "usgovtexas"
-      }
+  "$schema": "https://schema.management.azure.com/schemas/2019-04-01/deploymentParameters.json#",
+  "contentVersion": "1.0.0.0",
+  "parameters": {
+    "azureAdminPwd": {
+      "value": "GEN-PASSWORD"
+    },
+    "azureAdmin": {
+      "value": "GEN-UNIQUE"
+    },
+    "location": {
+      "value": "usgovtexas"
+    },
+    "azurermPackageBaseUri": {
+      "value": "https://devopsgallerystorage.blob.core.windows.net/packages/"
```

Comment:
if this is required, then set this as the defaultValue in the template and remove this from here

Reply:
If we set this as the default value in the template, the TTK throws best practice violation error for hard coded url. Should we do it still?